data/tmpl/utility/content5.html

```diff
@@ -13,7 +13,12 @@
       <main>
         <h1>Free to Roam</h1>
         <p>Take your history, bookmarks and passwords with you.</p>
-        <button class="button">Get Firefox on your phone.</button>
+        <a class="mobile" href="https://itunes.apple.com/us/app/apple-store/id989804926?pt=373246&mt=8&ct=firefox-onboarding-addon" target="_blank">
+          <img src="/data/media/appleStore.svg" />
+        </a>
+        <a class="mobile" href="https://play.google.com/store/apps/details?id=org.mozilla.firefox&referrer=utm_source=firefox-browser&utm_medium=Referral&utm_campaign=onboarding-addon" target="_blank">
+          <img src="/data/media/playStore.png" />
```

Comment:
For this, and the image before, specify at least the width and provide and `alt`